source/use-cases/local-stub.rst

```diff
@@ -102,25 +154,132 @@ server should give the same result as specifying it did above (with
-Unbound instance you configured (in this case ``127.0.0.1``):
+goes to the default resolver of the machine.
 
 .. code-block:: text
 
-    ;; SERVER: 127.0.0.1#53(127.0.0.1)
+    dig example.com
 
-Note that the "SERVER" section in the output from :command:`dig` should also
-contain the local IP address of our server.
+It should look the same as with 
+the ``127.0.0.1`` IP specified as we did earlier.
 
 .. code-block:: text
 
     ;; SERVER: 127.0.0.1#53(127.0.0.1)
 
-..
-    XXX IS UNBOUND PERSISTENT HERE?!
+.. note::
+
+    Unbound is not persistent at this point, and will not start up when your 
+    system does (and possibly "breaking" your internet). This is fixed by
+    restarting your Unbound upon reboot.
+
+Package manager
+^^^^^^^^^^^^^^^
+
+To make Unbound persistant between restarts, we need to add it to the systemd
+service manager, for which we'll need a service file. If you installed Unbound
+via the package manager, this service file is already created for you and the
+only thing that is missing, is it executing our own configuration file.
+
+To make sure we execute Unbound with our own configuration, we copy our config
+file to the default location of the config file:
+:file:`/etc/unbound/unbound.conf`. Make sure Unbound starts using the copied
+configuration (this can be done with the :option:`-c<unbound -c>` flag to
+specify the config location).
+
+Before you proceed to the next step, make sure to stop the Unbound that may 
+still be running. Now we can start our Unbound with systemd, which will restart
+automatically it if the system is rebooted.
+
+.. code-block:: text
+
+    systemctl start unbound
+
+To check that everything is correct, you can see the status (which should be 
+"active"):
+
+.. code-block:: text
+
+    systemctl status unbound
+
+We can now :command:`dig` a final time, to verify that this works
+
+
+Compilation
+^^^^^^^^^^^
+
+The steps for making Unbound persistant are almost exactly the same as if you
+installed it via the package manager, except that the service file that is 
+needed by systemd does not exist yet. So instead of changing it, we create it 
+and call it ``unbound.service``, and copy the minimally modified service file 
+supplied by the package manager. It should be located at: 
+``/lib/systemd/system/unbound.service``.
+
+So using your favorite text editor open the file:
+
+.. code-block:: bash
+
+    nano /lib/systemd/system/unbound.service
+
+and copy the file contents below:
+
+.. code-block:: text
+
+    [Unit]
+    Description=Unbound DNS server
+    Documentation=man:unbound(8)
+    After=network.target
+    Before=nss-lookup.target
+    Wants=nss-lookup.target
+
+    [Service]
+    Type=simple
+    Restart=on-failure
+    EnvironmentFile=-/usr/local/etc/unbound
+    ExecStart=/usr/local/sbin/unbound -d -p $DAEMON_OPTS
+    ExecReload=+/bin/kill -HUP $MAINPID
+
+    [Install]
+    WantedBy=multi-user.target
```

Comment:
Compiling the source code already generates an unbound.service file in `contrib/` with the correct paths. Copying that into `/lib/systemd/system/` would make more sense in case the user specified different paths than the ones shown above.

Reply:
Discussed that we're keeping this at this time